src/GraphQL/Types/Schema.cs

```diff
@@ -411,8 +412,41 @@ protected virtual void Validate()
             //TODO: add different validations, also see SchemaBuilder.Validate
             //TODO: checks for parsed SDL may be expanded in the future, see https://github.com/graphql/graphql-spec/issues/653
             // Do not change the order of these validations.
+            CoerceInputTypeDefaultValues();
             SchemaValidationVisitor.Instance.Run(this);
             AppliedDirectivesValidationVisitor.Instance.Run(this);
         }
+
+        protected virtual void CoerceInputTypeDefaultValues()
```

Comment:
It's `protected` so should have xml comments. Right?

Reply:
yes; done